### PR TITLE
[4.0] [a11y] Stats plugin

### PR DESCRIPTION
**Pull request by brianteeman**
Various a11y improvements

Change b to strong
Change a to button
Change text colour on button to pass color contrast checks

image

### CODE REVIEW

File: plugins/system/stats/layouts/field/uniqueid.php [hunk 1]
```diff
@@ -43,6 +43,6 @@
  */
 ?>
 <input type="hidden" name="<?php echo $name; ?>" id="<?php echo $id; ?>" value="<?php echo htmlspecialchars($value, ENT_QUOTES, 'UTF-8'); ?>">
-<a class="btn btn-secondary" id="js-pstats-reset-uid">
+<button class="btn btn-secondary" id="js-pstats-reset-uid">
```
Comment (C-Lodder) [inline plugins/system/stats/layouts/field/uniqueid.php]: `type` should be set to `button`

Comment (brianteeman) [reply plugins/system/stats/layouts/field/uniqueid.php]: Thanks @C-Lodder I always forget that

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
thanks

**Pull request merged**